gentoo-ci/gentoo-ci.bash

```diff
@@ -37,6 +37,11 @@ if [[ ${PREV_COMMIT} != ${CURRENT_COMMIT} ]]; then
 	"${PKGCHECK_RESULT_PARSER_GIT}"/pkgcheck2borked.py \
 		-x "${PKGCHECK_RESULT_PARSER_GIT}"/excludes.json \
 		-s -w -o warning.list *.xml
+
+	# Sort XML for better Git delta compression
+	xsltproc "${SCRIPT_DIR}"/sort-output.xsl output.xml >output.xml.tmp
```

Comment:
Actually, you can avoid the temporary file by piping the pkgcheck output straight to xsltproc above.

Reply:
done